cmd/plasmacli/flags/flags.go

```diff
@@ -0,0 +1,9 @@
+package flags
```

Comment:
this doesn't need to be its own package, IMO could just be with main.go or another file in a level up. Seems strange to me to have an entire package for one const declaration and an import

Reply:
i can't put it it in `main.go` because main packages cannot be imported. Placing it in `root.go` within `subcmd` causes a circular dependency. That's why i made it it's own package for now.

The latest sdk release has a home flag so this folder should be removed once we update